### PR TITLE
Query editor

### LINKED ISSUE #54: Allow to add custom query + editor
example: https://query.linkeddatafragments.org/

And be able to store it in your pod

### PR DESCRIPTION
**Pull request by axellorreyne**
Allow for creating new queries and edit existing queries, while still storing/syncing these to a pod

Fixes #54

### CODE REVIEW

File: src/components/Editor/QueryEditor.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            // Edit existing query
+        }
+    }
+
+    return (
+        <SimpleForm onSubmit={onSubmit}>
+            <TextInput source="id" name={"Query ID"} validate={[required()]} defaultValue={query ? query.id : undefined}/>
+            <TextInput source="name" name={"Name"} validate={[required()]} defaultValue={query ? query.name : undefined}/>
+            <TextInput source="description" name={"Description"} defaultValue={query ? query.description : undefined}/>
+            <TextInput source="query" multiline={true} name={"Query"} defaultValue={queryText}/>
```
Comment (axellorreyne) [inline src/components/Editor/QueryEditor.jsx:42]: While the _queryText_ is not undefined upon (re)rendering the editor component when editing an existing query, the text input field wont show it as a default value.
Instead it shows a blank field, as it it were undefined.

Comment (axellorreyne) [inline src/components/Editor/QueryEditor.jsx:42]: This should be some sort of rich text input, allowing for a cleaner, multiline editing of the query text

File: src/components/Editor/QueryEditor.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            ${queryText}
+            <TextInput source="truetext" multiline={true} name={"True text"} defaultValue={query ? (query.askQuery ? query.trueText : undefined) : undefined}/>
+            <TextInput source="falsetext" multiline={true} name={"False text"} defaultValue={query ? (query.askQuery ? query.falseText : undefined) : undefined}/>
```
Comment (axellorreyne) [inline src/components/Editor/QueryEditor.jsx:45]: All input fields should be properly styled/themed to be longer and not have the label/name overlap with the input content itself

Comment (mvanbrab) [reply src/components/Editor/QueryEditor.jsx:49]: At least the label/name overlap may be solved after merging the main branch. I saw that in the templated queries form too and solved it there. It was the login form's css that destroyed it as a side effect by the way.

**Pull request closed without merging**